### PR TITLE
[APR] fix -1 APR on exceptions

### PR DESCRIPTION
**Pull request by Rawallon**

### CODE REVIEW

File: apps/balancer-tools/src/app/apr/(utils)/calculatePoolStats.ts [hunk 1]
```diff
@@ -189,10 +189,10 @@ function calculateRoundAPR(
   const vebalAPR =
     balPriceUSD && tvl && votingShare
       ? ((WEEKS_IN_YEAR * (emissions * votingShare * balPriceUSD)) / tvl) * 100
-      : -1;
+      : null;
 
   return {
-    total: vebalAPR + feeAPR,
+    total: vebalAPR ? vebalAPR : 0 + feeAPR,
```
Comment (luizakp) [inline apps/balancer-tools/src/app/apr/(utils)/calculatePoolStats.ts:195]:
```suggestion
    total: vebalAPR ? vebalAPR + feeAPR : 0 + feeAPR,
```

**Comment by luizakp** [comment]
between 0 and 0 should be a line 🤔 

image

File: apps/balancer-tools/src/app/apr/(utils)/calculatePoolStats.ts [hunk 2]
```diff
@@ -189,10 +189,10 @@ function calculateRoundAPR(
   const vebalAPR =
     balPriceUSD && tvl && votingShare
       ? ((WEEKS_IN_YEAR * (emissions * votingShare * balPriceUSD)) / tvl) * 100
-      : -1;
+      : null;
 
   return {
-    total: vebalAPR + feeAPR,
+    total: vebalAPR ? vebalAPR + feeAPR : 0 + feeAPR,
```
Comment (ribeirojose) [inline apps/balancer-tools/src/app/apr/(utils)/calculatePoolStats.ts:195]:
```suggestion
    total: (vebalAPR || 0) + feeAPR,
```

File: apps/balancer-tools/src/app/apr/api/route.ts [hunk 3]
```diff
@@ -85,7 +85,10 @@ const computeAverages = (
       apr: {
         total: acc.apr.total + data.apr.total,
         breakdown: {
-          veBAL: acc.apr.breakdown.veBAL + data.apr.breakdown.veBAL,
+          veBAL:
+            data.apr.breakdown.veBAL != null
+              ? data.apr.breakdown.veBAL + acc.apr.breakdown.veBAL
+              : 0,
```
Comment (ribeirojose) [inline apps/balancer-tools/src/app/apr/api/route.ts:91]:
```suggestion
          veBAL:
            (data.apr.breakdown.veBAL || 0) + (acc.apr.breakdown.veBAL + 0)
```

**Pull request merged**